### DISCUSSION TITLE
I support OpenSnitch thanks to Python language !

### DISCUSSION THREAD

**Discussion by BobSquarePants** [evilsocket/opensnitch#894] [General]
Hi everyone,

First of all, to all the contributor of [opensnitch ](https://github.com/evilsocket/opensnitch) Good job !! Great piece of software !

I had myself some difficulty with it, but rest assure That I'm fully supporting this project !

Especially because it's mainly written in Python !! That have the best code readability ! and require way less typing/writing than any other programming language !

Cheers

**Comment by gustavo-iniguez-goya** [comment]
Thank you for the kind words @BobSquarePants :))

Thanks to bug reports and users' feedback, we keep improving day by day.
